### PR TITLE
Update RELEASE_NOTES for 1.247357

### PR DESCRIPTION
**Pull request by adam-mateen**
# Description of the issue
Update RELEASE_NOTES for 1.247357

# Description of changes
Update RELEASE_NOTES for 1.247357

# License
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

# Tests
none

# Requirements
_Before commit the code, please do the following steps._
1. Run `make fmt` and `make fmt-sh`
2. Run `make linter`

### CODE REVIEW

**Review by khanhntd — APPROVED** [review]
LGTM.

File: RELEASE_NOTES [hunk 1]
```diff
@@ -1,3 +1,14 @@
+========================================================================
+Amazon CloudWatch Agent 1.247357.0 (2022-12-15)
+========================================================================
+
+Bug fixes:
+* Update AWS SDK Go to fix mac build. (#640)
+* Fix control script starting CWA on Oracle Linux 9 by using systemctl (#645)
+* Fix handling negative statefile offset (#646)
+* Fix starting the agent with "auto" mode on an on premise host.
```
Comment (khanhntd) [inline RELEASE_NOTES:9]: **Nit**: PR  #652  should be included

Comment (adam-mateen) [reply RELEASE_NOTES:9]: Yes good point.

**Review by taohungyang — APPROVED** [review]
LGTM

**Review by khanhntd — APPROVED** [review]
LGTM.

**Pull request merged**